java/api/src/main/java/org/ray/api/function/RayFunc0.java

```diff
@@ -7,5 +7,6 @@
  */
 @FunctionalInterface
 public interface RayFunc0<R> extends RayFunc {
-  R apply();
+
+  R apply() throws Exception;
```

Comment:
I guess I missed something else, just wondering how does this relate to supporting void return?

Reply:
This is a small fix for another issue. Without this, Ray.call can't take a `RayFunc` with `throws Exception` declaration.

Reply:
I see, thanks.